docs/sources/enterprise/datasource_permissions.md

```diff
@@ -33,6 +33,8 @@ When permissions are enabled for a data source in an organization, you restrict
 
 <div class="clearfix"></div>
 
+> **Caution:** Enabling permissions for the default data source makes it unable to query for users not listed in the permissions. Panels using default data source will return `Access denied to datasource` error for that users.
```

Comment:
Unclear antecedent. What is the "it" that is unable to query? The data source? Grafana? Please clarify.